src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RC2/RC2CipherTests.cs

```diff
@@ -11,6 +11,7 @@ namespace System.Security.Cryptography.Encryption.RC2.Tests
     using RC2 = System.Security.Cryptography.RC2;
 
     [SkipOnMono("Not supported on Browser", TestPlatforms.Browser)]
+    [SkipOnMono("Android does not support RC2", TestPlatforms.Android)]
```

Comment:
Do we have a general rule for choosing between `PlatformSpecific` vs `SkipOnMono`? These seem kind of like lack of platform support more than mono support.

Reply:
I haven't seen any specific rules. I'll switch this to PlatformSpecific.